Source/SpatialGDK/Public/SpatialInterop.h

```diff
@@ -60,11 +60,12 @@ class FOutgoingReliableRPC
 };
 
 // Helper types used by the maps below.
-using FPendingOutgoingProperties = TPair<TArray<uint16>, TArray<uint16>>; // Pending incoming properties (replicated and handover).
-using FPendingIncomingProperties = TPair<TArray<const FRepHandleData*>, TArray<const FHandoverHandleData*>>;
+using FPendingOutgoingProperties = TPair<TArray<uint16>, TArray<uint16>>;
+using FPendingIncomingProperties = TPair<TMap<UObject*, TArray<const FRepHandleData*>>, TArray<const FHandoverHandleData*>>;  // Pending incoming properties (replicated and handover).
```

Comment:
This is a dense typedef, and though to parse. Worth a comment explaining the relationships between the types.
Or breaking the first map into it's own typedef.

Reply:
done